### PR TITLE
Reflection form

### LINKED ISSUE #5: User Input
As a user I should be able to fill out the text area and press submit.

### LINKED ISSUE #6: Reflection Post Request
Reflection should save to micro service.

### LINKED ISSUE #10: Create a microservice
We need a micro service that allows us to store and access user reflections.
The reflections should be an object containing an id, the date, the mood, and the body of the reflection.

### PR DESCRIPTION
**Pull request by nathanielmillard**
## Description
- connects our microservice to the app
- allows user emotion buttons to set state, and pass down through props
- passes functionality to set mood state down from app
- creates dynamic date for lobby
- sets state on ReflectionForm to save user reflection
- Build and imports POST request from apiCalls

## Motivation and Context
- closes #5
- closes #6
- closes #10

## How Has This Been Tested?
- [x] In console
- [x] In browser

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/Components/ReflectionForm/ReflectionForm.js [hunk 1]
```diff
@@ -1,17 +1,54 @@
...
+  render() {
+    return (
+      <section className='ReflectionForm'>
+      <form data-testid='reflectionForm'>
+      <h2  className='reflectionPrompt' data-testid='reflectionPrompt'> You're feeling fine, what went well today? </h2>
+      <label>
+      Write your reflection here:
+      <textarea
+      id='textArea'
+      data-testid='reflectionArea'
```
Comment (Kristiannmiller) [inline src/Components/ReflectionForm/ReflectionForm.js:33]: This testid comes in SO handy with testing. Glad we're implementing it!

File: src/Components/Lobby/Lobby.js [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
-      mood: '',
+
     }
   }
+
   render() {
     return (
       <section className='Lobby'>
-        <h1>Welcome Back, it's TODAY!</h1>
+        <h1>Welcome Back, it's {new Date().toDateString()}</h1>
```
Comment (BrigetteDoelp) [inline src/Components/Lobby/Lobby.js:17]: I'm glad we figured out a way to get a legible date into the site without having to implement moment. I know we could without too much trouble, but I like not having to mess with installing packages unless we absolutely have to.

File: src/Components/ReflectionForm/ReflectionForm.js (hunk 1 above)
Comment (BrigetteDoelp) [reply src/Components/ReflectionForm/ReflectionForm.js:33]: Agreed! Yay job shadows and getting tips and tricks from pros. I really feel like we've leveled up regarding streamlining our testing process.

**Comment by Kristiannmiller** [comment]
This looks great! I think we're in a great position to start working on getting the reflection list history page functionality working!

**Pull request merged**